bucket/rclone.json

```diff
@@ -15,9 +15,28 @@
             "extract_dir": "rclone-v1.57.0-windows-386"
         }
     },
-    "pre_install": "if (!(Test-Path \"$persist_dir\\rclone.conf\")) { New-Item -ItemType File \"$dir\\rclone.conf\" | Out-Null }",
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\rclone.conf\") -or !(Get-Item \"$persist_dir\\rclone.conf\").Length) {",
+        "   New-Item \"$persist_dir\\rclone.conf\" -ItemType File -Force | Out-Null",
+        "   if (Test-Path \"$env:APPDATA\\rclone\\rclone.conf\") {",
+        "       $importConfDir = \"$env:APPDATA\\rclone\"",
+        "   } elseif (Test-Path \"$env:XDG_CONFIG_HOME\\rclone\\rclone.conf\") {",
+        "       $importConfDir = \"$env:XDG_CONFIG_HOME\\rclone\"",
+        "   } else {",
+        "       if ($env:HOME) { $tilde = $env:HOME }",
+        "       elseif ($env:USERPROFILE) { $tilde = $env:USERPROFILE }",
+        "       else { $tilde = \"$env:HOMEDRIVE%\\$env:HOMEPATH\" }",
```

Comment:
Do you mean to use ternary operator to avoid the if blocks? Or you mean hoist just `$tilde` out and keep ifs? I'm not quite getting the "2nd nesting" part.

Reply:
Yes, declare $tilde outside, so that you don't have to write a nested if block.

Reply:
Can you show me how to do it?

Reply:
See 08ce9faff1f4c41fa65dbbda0878a862754ae453

Reply:
👍